06-building-a-program-start-to-finish/solving-the-problem/output/InvertedIndexClass.js

```diff
@@ -0,0 +1,133 @@
+/**
+ *
+ *
+ * @class InvertedIndex
+ */
+class InvertedIndex {
```

Comment:
Nice work. How about having another utility class that abstracts some of the helper methods you have here. InvertedIndex Class could have just the createIndex, getIndex and SearchIndex methods.